core/elements/ons-switch.es6

```diff
@@ -45,8 +45,30 @@ limitations under the License.
       this._compile();
       ModifierUtil.initModifier(this, scheme);
 
+      this._addAttributes();
```

Comment:
These are properties, not attributes, so please call the method `_addProperties`.